src/main/resources/collector-production.properties

```diff
@@ -0,0 +1 @@
+xroad-monitor-collector.shared-params-file=/etc/xroad/globalconf/FI/shared-params.xml
```

Comment:
How to switch between development and production modes? It would be nice to have this in `README.md` as well.